rules/windows/credential_access_kerberoasting_unusual_process.toml

```
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2020/11/02"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/02"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies network connections to Kerberos standard port and from an unusual process. On Windows only process that
+normally performs kerberos traffic from a Domain joined host is lsass.exe.
+"""
+false_positives = [
+    """
+    HTTP Traffic on Non Standard port 88, verify that the destination IP address is not related to a Domain Controller.
+    """,
+]
+from = "now-9m"
+index = ["logs-endpoint.events.*", "winlogbeat-*"]
+language = "eql"
+license = "Elastic License"
+name = "Kerberos Traffic from Unusual Process"
+risk_score = 43
+rule_id = "897dc6b5-b39f-432a-8d75-d3730d50c782"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+type = "eql"
+
+query = '''
+network where event.type == "start" and network.direction == "outgoing" and
+ destination.port == 88 and source.port >= 49152 and
+ process.executable != "C:\\Windows\\System32\\lsass.exe" and destination.address!="127.0.0.1" and destination.address!="::1" and
```

```suggestion
 process.executable != "C:\\Windows\\System32\\lsass.exe" and destination.address !="127.0.0.1" and destination.address !="::1" and
```
this isn't a big deal, just for consistency purposes, we should either use a space in all of them, or no space in all of them